### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by sumairq**
# Added the following

- Html for contact form.
- Styles for contact form.
- Background illustration for contact form.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,125 +1,172 @@
...
+              src="assets/images/social_media/linkedin.svg" alt="Linkedin icon"></a></li>
+        <li class="web_buttons_3"><a href="https://github.com/sumairq"><img src="assets/images/social_media/github.svg"
+              alt="Github icon"></a></li>
+      </ul>
+    </div>
+
+    <div class="aboutme_text">
+      <p>sumairqaisar992@gmail.com</p>
+    </div>
+  </section>
```
Comment (rotimiazeez) [inline index.html]: - [ ] Kindly wrap the `contact` section in a `footer` tag to make your code more readable, understandable and semantic 😉👍

File: index.html [hunk 2]
```diff
@@ -1,125 +1,173 @@
...
+        </li>
+        <li>
+          <input type="email" placeholder="Email address" name="email_address" required>
+        </li>
+        <li>
+          <textarea name="message" required maxlength="500">Enter text here</textarea>
+        </li>
+        <li>
+          <button class="form-button" type="submit">Get in touch</button>
+        </li>
```
Comment (rotimiazeez) [inline index.html]: - [ ] Please note that the form inputs are not a list items, your code will be more readable if you use👇
 ``` 
   <div id="contact-list">
     <input>
     <input>
     <textarea>
     <button></button>
     </div>
```

File: index.html [hunk 3]
```diff
@@ -1,125 +1,173 @@
...
+    <form action="https://formspree.io/f/meqnjpgd" method="post">
+      <ul id="contact-list">
+        <li>
+          <input type="text" maxlength="30" name="full_name" placeholder="Full name" required>
+        </li>
+        <li>
+          <input type="email" placeholder="Email address" name="email_address" required>
+        </li>
+        <li>
+          <textarea name="message" required maxlength="500">Enter text here</textarea>
```
Comment (rotimiazeez) [inline index.html]: - [ ] Kindly add required to the `textarea` to make sure it's validated 😉

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi Team👋, 

### Good job so far!👏
### There are some issues that you still need to work on to go to the next project but you are almost there!
> Kindly make the suggested changes and you'll be unstoppable🎉🎉✔
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by HENRYKC24 — APPROVED** [review]
## Hi @sumairq:wave:, 
# STATUS: APPROVED :heavy_check_mark: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**